tests/ops/api/v1/endpoints/test_privacy_request_endpoints.py

```diff
@@ -6114,6 +6116,54 @@ def test_create_privacy_request(
         assert len(response_data) == 1
         assert run_access_request_mock.called
 
+    @pytest.mark.parametrize(
+        "source, expected_submitted_by",
+        [
+            (PrivacyRequestSource.request_manager, lambda user: user.client.user_id),
+            (PrivacyRequestSource.privacy_center, lambda _: None),
+            (None, lambda _: None),
```

Comment:
We only want to store `submitted_by` if the privacy request is created from the Request manager page in the Admin UI